### ISSUE TITLE
[Console Wallet]make wallet exit on startup of wrong db existing version

### ISSUE THREAD

**Issue by SWvheerden** [tari-project/tari#5264]
Pending #5252 
The wallet can read the network version that last opened it up. 
If the network does not match, close the wallet with an appropriate error message.

**Comment by brianp** [comment]
#5252  was fixed with #5263. Is this issue still relevant?

**Comment by SWvheerden** [comment]
yes, this is for console wallet application it self, which does not check or use this yet

**Comment by SWvheerden** [comment]
We should probably check the type as well

**Comment by SWvheerden** [comment]
Is not important enough to work on atm, so closing this in the meantime
